gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/iceberg/BaseIcebergCatalog.java

```diff
@@ -39,7 +41,7 @@ protected BaseIcebergCatalog(String catalogName, Class<? extends Catalog> compan
   }
 
   @Override
-  public IcebergTable openTable(String dbName, String tableName) {
+  public IcebergTable openTable(String dbName, String tableName) throws IOException {
```

Comment:
now that we've separated the concerns of opening a table from verifying its existence (i.e. to support opening a table that doesn't yet exist)... aren't we able to remove this exception declaration?

Reply:
Yup... I have removed it in my latest commit!